tests/Yaapii.Atoms.Tests/IO/WriterAsOutputStreamTest.cs

```diff
@@ -0,0 +1,128 @@
+// MIT License
+//
+// Copyright(c) 2022 ICARUS Consulting GmbH
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+using System;
+using System.IO;
+using System.Text;
+using Xunit;
+using Yaapii.Atoms.Bytes;
+using Yaapii.Atoms.Enumerable;
+using Yaapii.Atoms.Text;
+
+namespace Yaapii.Atoms.IO.Tests
+{
+    public sealed class WriterAsOutputStreamTest
+    {
+        [Fact]
+        public void WritesContentToFile()
+        {
+            var dir = "artifacts/WriterAsOutputStreamTest"; var inputFile = "large-text.txt";
+            var inputPath = Path.GetFullPath(Path.Combine(dir, inputFile));
+            var outputFile = "text-copy.txt";
+            var outputPath = Path.GetFullPath(Path.Combine(dir, outputFile));
+
+            Directory.CreateDirectory(dir);
```

Comment:
This leaves the directory present after the test. We always want to cleanup after a test (and be sure that we do not test data from a previous test). Please have a look at the object TempDirectory and TempFile and use them here.